cdci_data_analysis/flask_app/dispatcher_query.py

```diff
@@ -809,10 +811,13 @@ def find_job_id_parameters(self, job_id):
         else:
             return json.load(open(scratch_dir_parameters[0]))
 
-
     # TODO make sure that the list of parameters to ignore in the frontend is synchronized
-    def generate_products_url_from_par_dict(self, products_url, par_dict) -> str:
+    def generate_products_url_from_par_dict(self, products_url, par_dict, include_use_scws=False) -> str:
         par_dict = par_dict.copy()
+        if include_use_scws:
```

Comment:
is it possible to determine if `include_use_scws` within this function, from the parameter list. If it is needed or not can be derived unambigiously, by convention: if scw list is provided, it is used. I think. That's what is provided to the API.